src/main/java/org/jabref/gui/importer/fetcher/WebSearchPane.java

```diff
@@ -74,6 +78,9 @@ protected Node createContentPane() {
         TextField query = SearchTextField.create();
         query.getStyleClass().add("searchBar");
 
+        // Weird issue: viewModel.queryValidationStatus().not() does not work...
+        BindingsHelper.includePseudoClassWhen(query, QUERY_INVALID, viewModel.queryValidationStatus().validProperty().not());
```

Comment:
@tobiasdiez Calling `not()` on a `ReadOnlyBooleanProperty` should negate the property. However, the property is always `true` then. Quick fix: Remove `not()` and have the property returning the **in**valid status. However, this approach does not work here as here `null` means `true` and a String object (i.e., the concrete error message)  means `false`. No explicit Boolean return value in our case. Do you have an idea how we could fix that?

Reply:
Probably a issue with the setup in this particular case, since validProperty.not is used quite often in our code base: https://github.com/JabRef/jabref/search?q=validProperty

Since this is an issue with the view model, you can easily write a unit test verifying that it works.

Reply:
> Probably a issue with the setup in this particular case, since validProperty.not is used quite often in our code base: [JabRef/jabref/search (validProperty)](https://github.com/JabRef/jabref/search?q=validProperty)

@calixtus Maybe this is "automagically" solved if we show the error in a tooltip / textbox.

> Since this is an issue with the view model, you can easily write a unit test verifying that it works.

Not as easy as writing a logic test, but should be doable when looking to existging `ModelTest` classes.

Reply:
Tests added, tests work... .not() not.

Reply:
What did you mean by `not()` not? Does `not()` works in the tests? If it works in the tests, did you tried binding in to a text field that prints TRUE/FALSE to make sure that the problem is not in the validation display?

Reply:
The tests, as I have introduced them yesterday, work. I did not have time to investigate further, since I had not too much time. But I'll try to get back on this later in the next few days.